File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.cs
using System;

public class Solution {
    public int solution(int n, string control) 
    {
        foreach(char c in control)
        {
            if(c == 'w')
                n += 1;
            else if(c == 's')
                n -= 1;
            else if(c == 'd')
                n += 10;
            else if(c == 'a')
                n -= 10;
        }
        return n;
    }
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.cpp
#include <string>
#include <vector>
#include <numeric>

using namespace std;

int solution(int n, string control) {

    for (char c : control)
    {
        if (c == 'w')
        {
           n += 1; 
        }
        else if (c == 's')
        {
           n -= 1; 
        }
        else if (c == 'd')
        {
           n += 10; 
        }
        else if (c == 'a')
        {
           n -= 10; 
        }
    }
    
    return n;
}